dataBase/DBCreator.py

```diff
@@ -7,10 +7,39 @@
 
 # table name to schema mapping
 Schemas= {   
+'housemls':
+	    '''
+	    houseID int primary key,
```

Comment:
houseId should not be primary key as we can have the same houseId listed with different mls.